src/conversions/unitconvert.cpp

```diff
@@ -252,21 +252,22 @@ static double yyyymmdd_to_mjd(double yyyymmdd) {
 /***********************************************************************/
 static double mjd_to_yyyymmdd(double mjd) {
 /************************************************************************
-    Date given in YYYY-MM-DD format.
+    Date returned in YYYY-MM-DD format.
 ************************************************************************/
-    double mjd_iter = 14 + 31;
-    int year = 1859, month=0, day=0;
+    unsigned int mjd_iter = 14 + 31;
```

Comment:
Perhaps rename this to `date_iter` instead. It doesn't make much sense now that the `mjd` variable is gone.

Reply:
Yep, that's a fair point.  Branch updated.